### PR TITLE
Update xvs icon

### PR DESCRIPTION
**Pull request by baomastr**

### CODE REVIEW

**Comment by therealemjy** [comment]
There's an issue with the `viewbox` of the new SVG icon, which makes it look smaller than the one of the VAI token:
Screenshot 2022-03-22 at 16 55 37

Screenshot 2022-03-22 at 16 55 40

**Comment by baomastr** [comment]
> There's an issue with the `viewbox` of the new SVG icon, which makes it look smaller than the one of the VAI token:

fixed with reimport token icon from svg instead of png

File: src/assets/img/venus_active.svg [hunk 1]
```diff
@@ -1,33 +1,3 @@
...
 </svg>
```
Comment (therealemjy) [inline src/assets/img/venus_active.svg]: Let's use camel case to name this file: `venusActive.svg`

**Review by coreyar — APPROVED** [review]

**Review by therealemjy — CHANGES_REQUESTED** [review]
There's a few files that need to be removed since they aren't used anymore:
- `src/assets/img/venus_32.png`
-  `src/assets/img/coins/xvs.png`

**Comment by baomastr** [comment]
> There's a few files that need to be removed since they aren't used anymore:

done

**Comment by baomastr** [comment]
@therealemjy 
Designer asked to hold the PR until the business team approved changes, so I added WIP status

**Comment by coreyar** [comment]
> @therealemjy Designer asked to hold the PR until the business team approved changes, so I added WIP status

Let's use Draft PRs instead of the PR title to note that a PR is still in progress

**Review by therealemjy — APPROVED** [review]

**Pull request merged**